src/frontend/src/utils/authenticatorAttachment.ts

```diff
@@ -0,0 +1,21 @@
+import { KeyType } from "../../generated/internet_identity_types";
+
+/**
+ * Converts a WebAuthn authenticator attachment to an II key type
+ */
+export function authenticatorAttachmentToKeyType(
+  authenticatorAttachment: AuthenticatorAttachment | undefined
+): KeyType {
+  let keyType: KeyType = { unknown: null };
+  switch (authenticatorAttachment) {
+    case "cross-platform":
+      keyType = { cross_platform: null };
+      break;
+    case "platform":
+      keyType = { platform: null };
+      break;
+    default:
+      break;
+  }
+  return keyType;
```

Comment:
do you think something like this would be simpler to understand?

```ts

if(authenticatorAttachment === "cross-platform") {
  return { cross_platform: null };
} else if (authenticatorAttachment === "platform") {
  return { platform: null };
} else {
  return undefined;
}
```